libraries/AC_PID/AC_P_2D.cpp

```diff
@@ -21,16 +21,15 @@ AC_P_2D::AC_P_2D(float initial_p) :
 }
 
 // update_all - set target and measured inputs to P controller and calculate outputs
-Vector2f AC_P_2D::update_all(postype_t &target_x, postype_t &target_y, const Vector2f &measurement)
+Vector2f AC_P_2D::update_all(Vector2p target, const Vector2p &measurement)
```

Comment:
Should this be a reference like, "Vector2p& target"?  I think we need this so that we can update the target when it's dragged by the current estimate

Reply:
Fixed.